classes/DefaultArgumentRemoverReturnTypes100.php

```diff
@@ -73,7 +73,7 @@ public function toString() : string
     private function removeDefaultArguments(Invocation $invocation, string $class)
     {
         $remover = function () {
-            MockFunctionGenerator::removeDefaultArguments($this->parameters);
+            MockFunctionGenerator::removeDefaultArguments($this->parameters());
```

Comment:
fix only vars can be passed as reference.

```suggestion
            $paramaters = $this->parameters();
            MockFunctionGenerator::removeDefaultArguments($paramaters);
```